### ISSUE TITLE
Write docstrings

### ISSUE THREAD

**Issue by whitenoiseoss** [evennia/ainneve#2]
Every method and class that is written for Ainneve should have a docstring in the Google style.

Examples can be found here: http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html
This work will coincide with Griatch's work here: https://groups.google.com/forum/#!topic/evennia/1t2mZSc0K1E

This issue will be ongoing.

**Comment by philygee** [comment]
Hey there, are you still looking for help with docstrings? I'd like to help!

**Comment by aliceafterall** [comment]
Absolutely, that would an excellent thing. Please build against develop - the master branch is not updated for 0.7 yet!
